Runtime/System/ClientSimMain.cs

```diff
@@ -468,8 +480,12 @@ private void RemoveSDKLinks()
             VRCStation.exitStationDelegate -= ClientSimStationHelper.ExitStation;
             VRCPlayerApi._UseAttachedStation -= ClientSimStationHelper.UseAttachedStation;
 
-            VRC_UiShape.GetEventCamera -= _player.GetCameraProvider().GetCamera;
-            
+            // Player can be invalid if we weren't able to spawn
+            if (_player)
+            {
+                VRC_UiShape.GetEventCamera -= _player.GetCameraProvider().GetCamera;
+            }
+
```

Comment:
I added this specifically because _player is invalid at this point and was throwing an error in the console which was a distraction from the main issue (ClientSim failed to initialize, player failed to spawn). I don't understand what you mean by "added to the area adding the api", can you rephrase that?

Reply:
It's probably not an issue, but I was referring to this:
https://github.com/vrchat-community/ClientSim/blob/042856263bd6fc0835e50eabd6822a1dbc7a067c/Runtime/System/ClientSimMain.cs#L354

Reply:
Thanks for the clarification. It's not specifically an issue in the place you mention because the enclosing function `SetupSDKLinks` doesn't run if ClientSim fails to initialize, but RemoveSDKLinks does. However, I think they should be matched so I'll push an update with a null check before this call as well.